rasa/utils/common.py

```diff
@@ -200,10 +208,14 @@ def not_found():
         else:
             raise ValueError(f"Configuration '{name}' key not found.")
 
-    if not os.path.exists(GLOBAL_USER_CONFIG_PATH):
+    # need to use `rasa.constants.GLOBAL_USER_CONFIG_PATH` to allow patching
+    # in tests
+    config_path = rasa.constants.GLOBAL_USER_CONFIG_PATH
+
+    if not os.path.exists(config_path):
```

Comment:
Can't we skip the the `os.path.exist` check and check whether we got an empty config or not?

Reply:
it is not the same, the file might exist but be empty. the logic was like that before, I don't think we should change it